src/libs/ReportUtils.js

```diff
@@ -4097,26 +4097,26 @@ function getTaskAssigneeChatOnyxData(accountID, assigneeAccountID, taskReportID,
 }
 
 /**
- * Returns an array of the participants Ids of a report
+ * Returns an array of the visible member Ids of a report
  *
  * @param {Object} report
  * @returns {Array}
  */
-function getParticipantsIDs(report) {
+function getVisibleMembersIDs(report) {
     if (!report) {
         return [];
     }
 
-    const participants = report.participantAccountIDs || [];
+    const visibleChatMembersIDs = report.visibleChatMemberList || [];
```

Comment:
So, we now have `report.participantAccountIDs` and `visibleChatMemberList`? 

The latter should maybe be called `visibleChatMemberAccountIDs`. 

Separately, from discussing some things with @iwiznia related to the Group chats design doc... perhaps we should have modified things so that we eventually have something more flexible like:

```js
report.participants = {
    [accountID1]: {visible: false, role: 'member'},
    [accountID2]: {visible: true, role: 'admin'},
}
```

Reply:
I am opening the PRs in Auth and Web-E for renaming this.

Reply:
Maybe instead of visible it would be the notification preference of each participant? Would that be useful?

Reply:
Hmm, sorry I don't want to give the wrong idea here about the second suggestion. Let's talk about it some before raising any PRs?

Reply:
For sure, I wasn't planning working on that right now